packages/vite-plugin-web-extension/src/extension-runner/web-ext-runner.ts

```diff
@@ -27,9 +27,12 @@ export function createWebExtRunner(
 
       const initialConfig = await loadConfig({ pluginOptions, logger, paths });
       const target =
-        pluginOptions.browser === null || pluginOptions.browser === "firefox"
-          ? null
-          : "chromium";
+        pluginOptions.browser === "chrome"
+          ? "chromium"
+          : pluginOptions.browser === null ||
```

Comment:
I wonder if we should deprecate the `browser: null` setting. It's a bit cryptic 😅

Reply:
Huh, yeah, wonder what I was thinking lol. Let's leave it for now.